src/api/methods/define/Define.ts

```diff
@@ -231,6 +231,12 @@ function buildUrimapRequestBody(parms: IURIMapParms, usage: "server" | "client"
         requestAttrs.webservice = parms.webserviceName;
     }
 
+    if (parms.enable === false) {
+        requestAttrs.status = "DISABLED";
+    } else {
+        requestAttrs.status = "ENABLED";
+    }
```

Comment:
Should we evaluate if `parms.enable === true` as well?
Just wondering if we are forcing the users to have their URIMap resource be enabled by default if they don't specify it, meaning we are overwriting whatever behavior they have defined on their system.
I don't believe this is really an issue though, until a customer has a use-case for a recently defined URIMap to not be enabled on installation by default.
No Change Requested!

Reply:
I believe that the API sets it to be enabled by default, but I am not entirely certain. Now that I look at this a little more, I do not believe that the else is required.